### PR TITLE
🚀Add Custom Error Func for CSRF Middleware

### PR DESCRIPTION
**Pull request by hinoguma**
#1083

### CODE REVIEW

File: middleware/csrf/config.go [hunk 1]
```diff
@@ -89,6 +94,12 @@ var ConfigDefault = Config{
 	CookieSameSite: "Strict",
 	Expiration:     1 * time.Hour,
 	KeyGenerator:   utils.UUID,
+	ErrorHandler:   DefaultErrorHandler,
+}
+
+// Default ErrorHandler that process return error from fiber.Handler
+var DefaultErrorHandler = func(c *fiber.Ctx, err error) error {
```
Comment (kiyonlin) [inline middleware/csrf/config.go:101]: It's better to make `DefaultErrorHandler` private.

File: middleware/csrf/csrf_test.go [hunk 2]
```diff
@@ -233,3 +233,54 @@ func Test_CSRF_From_Cookie(t *testing.T) {
...
+	})
+
+	h := app.Handler()
+	ctx := &fasthttp.RequestCtx{}
+
+	// Empty/invalid CSRF token
+	ctx.Request.Reset()
+	ctx.Response.Reset()
+	ctx.Request.Header.SetMethod("POST")
+	ctx.Request.Header.Set("X-CSRF-Token", "johndoe")
```
Comment (kiyonlin) [inline middleware/csrf/csrf_test.go:282]: This case does not set a invalid cookie. And `empty` also means `without cookie` right?

**Review by kiyonlin — CHANGES_REQUESTED** [review]

File: middleware/csrf/config.go (hunk 1 above)
Comment (hinoguma) [reply middleware/csrf/config.go]: @kiyonlin 
Thank you for review.
I fixed. 1ad5a618cb0da239c41a218d3b06029556e7e571

File: middleware/csrf/csrf_test.go (hunk 2 above)
Comment (hinoguma) [reply middleware/csrf/csrf_test.go]: > This case does not set a invalid cookie.

I fixed Test_CSRF_ErrorHandler_InvalidToken for comparison between generated uuid token and invalid token.

> And empty also means without cookie right?

Empty means that token is not exists in request.
And I add Test_CSRF_ErrorHandler_EmptyToken in 1ad5a618cb0da239c41a218d3b06029556e7e571.


Test_CSRF_ErrorHandler_InvalidToken for
https://github.com/gofiber/fiber/blob/19e6a4429d1f9b6ae5f818c4b66a7357c78e58e4/middleware/csrf/csrf.go#L88-L103

Test_CSRF_ErrorHandler_EmptyToken for
https://github.com/gofiber/fiber/blob/19e6a4429d1f9b6ae5f818c4b66a7357c78e58e4/middleware/csrf/csrf.go#L82-L86

**Review by kiyonlin — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**